source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,498 @@
+
+ .. command-block::
+    :no-exec:
+
+    mv rep-seqs-dada2.qza rep-seqs.qza
+    mv table-dada2.qza full-table.qza
+
+
+Feature table and representative sequence summaries
+---------------------------------------------------
+
+After the quality filtering step completes, you'll want to explore the resulting data. You can do this using the following two commands, which will create visual summaries of the data. The ``feature-table summarize`` command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. The ``feature-table tabulate-seqs`` command will provide a mapping of feature IDs to sequences, and provide links to easily BLAST each sequence against the NCBI nt database. The latter visualization will be very useful later in the tutorial, when you want to learn more about specific features that are important in the data set.
+
+.. command-block::
+
+   qiime feature-table summarize \
+     --i-table full-table.qza \
+     --m-sample-metadata-file sample-metadata.tsv \
+     --o-visualization full-table.qzv
+   qiime feature-table tabulate-seqs \
+     --i-data rep-seqs.qza \
+     --o-visualization rep-seqs.qzv
+
+Generate a tree for phylogenetic diversity analyses
+---------------------------------------------------
+
+QIIME supports several phylogenetic diversity metrics, including Faith's Phylogenetic Diversity (alpha diversity) and weighted and unweighted UniFrac (beta diversity). In addition to counts of features per sample (i.e., the data in the ``FeatureTable[Frequency]`` QIIME 2 artifact), these metrics require a rooted phylogenetic tree relating the features to one another. This information will be stored in a ``Phylogeny[Rooted]`` QIIME 2 artifact. To generate a phylogenetic tree we will use ``sepp`` pipeline from the ``q2-fragment-insertion`` plugin. Please review the `q2-fragment-insertion`_ installation instructions.
+
+The pipeline uses the ``sepp`` program will create a rooted tree with the new sequences inserted in the backbone tree, by default the GreenGenes 99% similarity tree; the details of the placements are going to be saved in ``insertion-placements.qza``
+
+.. command-block::
+
+   qiime fragment-insertion sepp \
+     --i-representative-sequences rep-seqs.qza \
+     --o-tree insertion-tree.qza \
+     --o-placements insertion-placements.qza
+
+Not all fragments will be able to be inserted and further down analyses rely on having all features in the tree so we will need to filter our feature table. The filter step in the `q2-fragment-insertion` will generate a feature table with the fragments that are present in the tree (``table.qza``) and another one without them (``removed-table.qza``). We will summarize the these two new tables for inspection.
+
+.. command-block::
+
+     qiime fragment-insertion filter-features \
```

Comment:
the indentation is incorrect in this command block. It is causing these commands to display incorrectly, and is causing the next section of text to appear in the command block!

Reply:
good catch!